src/v/coproc/pacemaker.cc

```diff
@@ -55,7 +57,7 @@ pacemaker::pacemaker(ss::socket_address addr, ss::sharded<storage::api>& api)
 
 ss::future<> pacemaker::start() {
     _offs.timer.set_callback([this] {
-        (void)ss::with_gate(_offs.gate, [this] {
+        (void)ss::with_gate(_gate, [this] {
```

Comment:
what's the difference between _gate and _offs.gate

Reply:
did _offs.gate get deleted? it's not clear how we can just switch over to a differnet gate.

Reply:
no its just a rename. Before it logically only was connected to logic working with offset management, so i just renamed it.